pkg/ovs/ovn-nb-logical_switch_port.go

```diff
@@ -185,6 +185,41 @@ func (c *OVNNbClient) CreateVirtualLogicalSwitchPorts(lsName string, ips ...stri
 	return nil
 }
 
+// CreateVirtualLogicalSwitchPort create one virtual type logical switch for aap
+func (c *OVNNbClient) CreateVirtualLogicalSwitchPort(lspName, lsName, ip string) error {
+	exist, err := c.LogicalSwitchPortExists(lspName)
+	if err != nil {
+		klog.Error(err)
+		return err
+	}
+
+	// ignore
+	if exist {
+		return nil
+	}
+
+	lsp := &ovnnb.LogicalSwitchPort{
+		UUID: ovsclient.NamedUUID(),
+		Name: lspName,
+		Type: "virtual",
+		Options: map[string]string{
+			"virtual-ip": ip,
+		},
+	}
+
+	op, err := c.CreateLogicalSwitchPortOp(lsp, lsName)
+	if err != nil {
+		klog.Error(err)
+		return err
+	}
+
+	if err := c.Transact("lsp-add", op); err != nil {
+		return fmt.Errorf("create virtual logical switch ports for logical switch %s: %v", lsName, err)
```

Comment:
return fmt.Errorf("create virtual logical switch port %s for logical switch %s: %v", lspName, lsName, err)